src/main/java/uk/gov/hmcts/reform/divorce/caseformatterservice/mapper/DocumentCollectionDivorceFormatMapper.java

```
@@ -23,19 +32,32 @@
     private DocumentUrlRewrite documentUrlRewrite;
 
     @Mapping(source = "value.documentLink.documentFilename", target = "fileName")
-    @Mapping(source = "value.documentDateAdded", dateFormat = SIMPLE_DATE_FORMAT, target = "createdOn")
     @Mapping(source = "value.documentLink.documentUrl", target = "fileUrl")
     public abstract UploadedFile map(CollectionMember<Document> documentCollectionMember);
 
 
     @AfterMapping
     protected void mapDocumentIDToDocumentObject(CollectionMember<Document> document,
-                                                 @MappingTarget  UploadedFile result ) {
+                                                 @MappingTarget  UploadedFile result) {
 
         Optional.of(document)
             .map(documentEntry -> document.getValue())
             .map(Document::getDocumentLink)
             .map(DocumentLink::getDocumentUrl)
             .ifPresent(url -> documentUrlRewrite.getDocumentId(url).ifPresent(result::setId));
     }
+
+    @AfterMapping
+    protected void mapCreatedDateCheckEmpty(CollectionMember<Document> document,
+                                            @MappingTarget  UploadedFile result) {
+        try {
+            if (document.getValue().getDocumentDateAdded() != null && !document.getValue().getDocumentDateAdded().isEmpty()) {
+                SimpleDateFormat dateFormat = new SimpleDateFormat(SIMPLE_DATE_FORMAT);
+                Date date = new Date(dateFormat.parse(document.getValue().getDocumentDateAdded()).getTime());
+                result.setCreatedOn(date);
+            }
+        } catch (ParseException ex) {
+            log.warn("Unable to parse a date string on document");
```

Don't swallow this exception, if is the wrong format better propagate it

---

Would rather catch and then log it, so we can debug if required. Given the parser may throw this, it requires a try / catch